### PR TITLE
Merges tgui changes from tgchat PR

### PR DESCRIPTION
**Pull request by ThePotato97**
## About the PR 

Merges changes from the tgchat PR tgui

## Why's this needed? 

Cause it is

## Changelog

```
(u)ThePotato97:
(*)Updated tgui to a newer version (fixes input boxes)
```

### CODE REVIEW

File: tgui/docs/converting-old-tgui-interfaces.md [hunk 1]
```diff
@@ -0,0 +1,332 @@
+# Converting old tgui interfaces to tgui-next
```
Comment (ZeWaka) [inline tgui/docs/converting-old-tgui-interfaces.md:1]: not needed, delete this file

File: tgui/docs/migration-to-v4-from-v3.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+# tgui Migration Guide to v4 from v3
```
Comment (ZeWaka) [inline tgui/docs/migration-to-v4-from-v3.md:1]: not needed, delete this file

File: tgui/docs/tutorial-and-examples.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ ui_state()
 - `src_object` - The atom, which UI corresponds to in the game world.
 - `ui_interact` - The proc where you will handle a request to open an
 interface. Typically, you would update an existing UI (if it exists),
-or set up a new instance of UI by calling the `tgui_process` process.
+or set up a new instance of UI by calling the `SStgui` subsystem.
```
Comment (ZeWaka) [inline tgui/docs/tutorial-and-examples.md:17]: revert your changes here

File: tgui/packages/tgui/styles/interfaces/NuclearBomb.scss [hunk 4]
```diff
@@ -0,0 +1,64 @@
+@use '../base.scss';
```
Comment (ZeWaka) [inline tgui/packages/tgui/styles/interfaces/NuclearBomb.scss:1]: delet this

File: tgui/packages/tgui/styles/interfaces/Roulette.scss [hunk 5]
```diff
@@ -0,0 +1,100 @@
+@use '../base.scss';
```
Comment (ZeWaka) [inline tgui/packages/tgui/styles/interfaces/Roulette.scss:1]: delet this

File: tgui/docs/converting-old-tgui-interfaces.md (hunk 1 above)
Comment (ZeWaka) [inline tgui/docs/converting-old-tgui-interfaces.md:1]: delet this

File: tgui/docs/migration-to-v4-from-v3.md (hunk 2 above)
Comment (ZeWaka) [inline tgui/docs/migration-to-v4-from-v3.md:1]: delet this

File: tgui/docs/tutorial-and-examples.md [hunk 6]
```diff
@@ -14,7 +14,7 @@ ui_state()
 - `src_object` - The atom, which UI corresponds to in the game world.
```
Comment (ZeWaka) [inline tgui/docs/tutorial-and-examples.md:14]: revert this

File: tgui/packages/tgui/webpack.config.js [hunk 7]
```diff
@@ -53,7 +57,7 @@ module.exports = (env = {}, argv) => {
                   ['@babel/preset-env', {
                     modules: 'commonjs',
                     useBuiltIns: 'entry',
-                    corejs: '3.6',
+                    corejs: '3',
```
Comment (ZeWaka) [inline tgui/packages/tgui/webpack.config.js]: my dude?

**Comment by ZeWaka** [comment]
who needs mordent 😸

**Comment by mordent-goonstation** [comment]
> who needs mordent 😸

Joke's on you, you got tricked into doing my work.

**Pull request merged**